tools/biosnoop.py

```diff
@@ -206,10 +206,13 @@ def print_event(cpu, data, size):
 
     delta = float(event.ts) - start_ts
 
+    disk_name = event.disk_name.decode('utf-8', 'replace')
+    if not disk_name:
+        disk_name = 'unknown'
```

Comment:
the same here, please use "\<unknown\>".

Reply:
done